src/app/components/dashboard/Dashboard.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import config from '../../config';
+import { SUCCESS } from '../../constants/HTTPStatus';
+
+import ServicesSummary from './services-summary/ServicesSummary';
+import SummaryPieChart from './charts/SummaryPieChart';
+import Logs from './logs/Logs';
+import ServicesTable from './table/ServicesTable';
+
+
+class Dashboard extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      services: [],
+      totalRunning: 0,
+      totalStopped: 0
+    };
+  }
+
+  componentWillMount() {
+    axios.get(`${config.API_ENDPOINT}/api/status`).then((result) => {
```

Comment:
This API logic should go into services.

Create a new service something like `status.js` and have a function `fetchStatus()` and use it here.